src/main/java/de/oliver/fancyholograms/commands/HologramCMD.java

```diff
@@ -209,7 +209,8 @@ public boolean execute(@NotNull CommandSender sender, @NotNull String label, @No
                     yield FancyNpcsPlugin.get().getNpcManager().getAllNpcs().stream().map(npc -> npc.getData().getName());
                 }
                 case "block" -> Arrays.stream(Material.values()).filter(Material::isBlock).map(Enum::name);
-                case "seethrough", "visiblebydefault" -> Stream.of("true", "false");
+                case "seethrough" -> Stream.of("true", "false");
+                case "visiblebydefault" -> new VisibleByDefaultCMD().tabcompletion(sender, hologram, args).stream();
```

Comment:
Creating a new command object for every tabcomplete is probably not a good idea, I would just copy the contents of the method over.